
Although now removed in gnulib:
	https://github.com/coreutils/gnulib/commit/329877ac214ca86f7adb6a59f
sed has not yet updated.

diff -wpruN --no-dereference '--exclude=*.orig' a~/gnulib-tests/localename.c a/gnulib-tests/localename.c
--- a~/gnulib-tests/localename.c	1970-01-01 00:00:00
+++ a/gnulib-tests/localename.c	1970-01-01 00:00:00
@@ -51,7 +51,7 @@
 # if defined __sun
 #  if HAVE_GETLOCALENAME_L
 /* Solaris >= 12.  */
-extern char * getlocalename_l(int, locale_t);
+extern const char * getlocalename_l(int, locale_t);
 #  elif HAVE_SOLARIS114_LOCALES
 #   include <sys/localedef.h>
 #  endif
